src/client/sandbox/code-instrumentation/location/index.js

```diff
@@ -34,11 +37,17 @@ export default class LocationAccessorsInstrumentation extends SandboxBase {
 
         window[INSTRUCTION.setLocation] = (location, value) => {
             if (isLocation(location)) {
-                location = value;
+                var resourceType = null;
 
-                return location;
-            }
+                if (window.self !== window.top) {
```

Comment:
BTW, it would be nice to have eslint rule for this. @LavrovArtem can you take care of this, please?